internal/terraform/local.go

```diff
@@ -44,13 +45,16 @@ func (l *local) Prepare() error {
 	var (
 		tfpath        = "/usr/local/bin/terraform"
```

Comment:
@psihachina Can you please clarify, are we still installing terraform into that location? If so, this is not what we want. This path should not be used, and terraform should be executed only from ~/.ize location in my opinion.

Reply:
@AutomationD No, we are not installing terraform to this location, but I will delete this variable in another story about refactoring this package so as not to break anything